cmd/localCI/Makefile

```diff
@@ -0,0 +1,24 @@
+DESTDIR :=
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
+SYSCONFDIR := $(PREFIX)/etc
+LIBEXECDIR := $(PREFIX)/libexec
+
+SOURCES := $(shell find . 2>&1 | grep -E '.*\.(c|h|go)$$')
```

Comment:
It's worth noting that `go build` will essentially do the same as the `find` though: if you create a new `.go` file as an experiment, `go build` **will** try to build it ;-)